doc/system_description/fpga.md

```diff
@@ -1,5 +1,8 @@
 # The TKey FPGA
 
+**NOTE:** Documentation migrated to dev.tillitis.se, this is kept for
```

Comment:
The FPGA document hasn't been moved to the Dev Handbook either. Maybe it should stay?

Reply:
Good catch. Will remove the note.